README.md

```diff
@@ -70,6 +70,13 @@ You can find several examples in the `examples` directory, such as:
 ## Upgrading version and migrating DB
 The `install.lock` file is located inside the container volume `/var/www/documents`.
 
+## Early support for PostgreSQL
+Setting `DOLI_DB_TYPE` to `pgsql` enable Dolibarr to run with a PostgreSQL database.
+When set to use `pgsql`, Dolibarr must be installed manually on it's first execution:
+ - Browse to `http://0.0.0.0/install`;
+ - Follow the installation setup;
+ - Add `install.lock` inside the container volume `/var/www/html/documents` (ex `docker-compose exec services-data_dolibarr_1 /bin/bash -c "touch /var/www/html/documents/install.lock"`).
+
```

Comment:
- put it just before "Environment variables summary" paragraph
- the user must also change the permission of the conf file

Reply:
- You mean to put the "Early support for PostgreSQL" just before "Environment variables summary"? If so it already is there
- The conf file is already managed in the docker-run.sh (line 84-88)
```bash
if [[ ${DOLI_DB_TYPE} == "pgsql" ]]; then
  chmod 600 /var/www/html/conf/conf.php
else
  chmod 400 /var/www/html/conf/conf.php
fi
```